packages/openapi3/test/primitive-types.test.ts

```diff
@@ -189,4 +189,56 @@ describe("openapi3: primitives", () => {
       });
     });
   });
+
+  describe("date format", () => {
+    it("set format to 'date-time' by default", async () => {
```

Comment:
Should have an alternate where this is ued in a header, and the default format is rfc-7231